calebgmahala/SCLR
# [Resolves #18] Render Levels

# [Resolves #18] Render Levels
Add render levels to entities

## Changes
- Added layer property to Entity
- Added checks to World draw methods to check layer property
- Updated demo to test for new layer changes

## Test
- Run application in development mode
- Use tunnel placer to place tunnel
- Navigate User over tunnel and notice that tunnel renders over User
